### ISSUE TITLE
iOS app does not run with latest Xcode and Mac version

### ISSUE THREAD

**Issue by rangarajanps** [londonappbrewery/xylophone-flutter#21]
Build failed with multiple link error like below,
ld: warning: Could not find or use auto-linked library 'swiftCoreAudio'

 I resolved the issue by following the suggestion in https://stackoverflow.com/questions/52536380/why-linker-link-static-libraries-with-errors-ios 

I opened Runner.xcodeproj in Xcode and it asked to update the Deployment models, which I clicked "Continue" and then created an empty File.Swift in Runner directory in Xcode which created a Bridging Header. 

Please update the base project so that it is easy for learners to start using this project.

**Comment by bitpoetics** [comment]
@rangarajanps It works! Thanks 😄

**Comment by saan-saeteurn** [comment]
> @rangarajanps It works! Thanks 😄

Agreed, this also helped me as well.  Open the iOS project and create a new Swift file.  When prompted for app target, make sure to select your 'xylophone-flutter' project and then select Bridging Header.  Lastly, build it in Xcode and it should build successfully.  Now you should be able to run your Dart project in Android Studio.
